ci-operator/step-registry/quay-tests/test-quay-dast/quay-tests-test-quay-dast-commands.sh

```
@@ -2,87 +2,34 @@
 
-if [ ${QUAY_ENV} = 'QUAY' ]; then
-   echo "Generating Quay OpenAPI File..."
-   QUAY_URL=$(cat "$SHARED_DIR"/quayroute)
-   QUAY_SHORT_NAME="quay"
-   QUAY_ACCESS_TOKEN=$(cat "$SHARED_DIR"/quay_oauth2_token)
-   QUAY_OAUTH2_TOEKN="Bearer $QUAY_ACCESS_TOKEN"
-   cat "$SHARED_DIR"/quay_api_discovery | jq > openapi.json && cp openapi.json $ARTIFACT_DIR || true 
-fi
+bash generate-quay-config ./quay-credentials.yaml
 
+cp *-scan.yaml "${ARTIFACT_DIR}/" 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)